nvflare/dashboard/application/blob.py

```diff
@@ -324,8 +306,7 @@ def gen_user(key, id):
     else:
         overseer_agent = {"path": "nvflare.ha.dummy_overseer_agent.DummyOverseerAgent"}
         overseer_agent["args"] = {"sp_end_point": f"{project.server1}:8002:8003"}
-    # agent_config = {"overseer_agent": overseer_agent}
-    config["admin"]["overseer_agent"] = overseer_agent
+    config["admin"].update({"overseer_agent": overseer_agent})
```

Comment:
How can this fix #1219 ?

I thought the missing fields are not "overseer_agent"?

Reply:
config is created from fed_admin section of master_template.yml.  Its admin contains all information, except the overseer_agent.  Therefore, updating the config["admin"] dictionary with {"overseer_agent": overseer_agent} will add that, while keeping original info unchanged.